ml/backend/ggml/ggml.go

```diff
@@ -299,10 +302,15 @@ func New(r *os.File, params ml.BackendParams) (ml.Backend, error) {
 
 	// concurrently read in tensor data. uses a section reader which is safe for concurrent reads
 	sr := io.NewSectionReader(r, int64(meta.Tensors().Offset), n-int64(meta.Tensors().Offset))
-	var g errgroup.Group
+
+	var doneCount atomic.Uint32
+
+	g, ctx := errgroup.WithContext(context.Background())
```

Comment:
As a general pattern, we might want to make `New` take a context to cancel loading. We wouldn't particularly do anything with it now but it might be useful depending on how the runner evolves and it could be easier to add it now rather than later as we have more backends.